runbooks/workflow/payments/add_agency_dashboard_data_source.md

```diff
@@ -0,0 +1,185 @@
+# Metabase: Configure Agency Payments Data and Generate Dashboards
+
+This documentation is broken out into two sections:
+
+- [Add a New Agency Data Source to Metabase and Create Permissions](#add-a-new-agency-data-source-to-metabase-and-create-permissions)
+- [Create a New Agency Dashboard and the Comprising Questions](#create-a-new-agency-dashboard-and-the-comprising-questions)
+
+## Add a New Agency Data Source to Metabase and Create Permissions
+
+As new agencies are introduced to the contactless payments program we will need to access their data within Metabase for use in their payments dashboard and other analysis. Because we use a [row access policy](https://github.com/cal-itp/data-infra/blob/main/warehouse/macros/create_row_access_policy.sql) in the warehouse code to limit access to data to authorized parties this is a multi-step process.
```

Comment:
```suggestion
As new agencies are introduced to the contactless payments program we will need to access their data within Metabase for use in their payments dashboard and other analysis. Because we use a [row access policy](https://cloud.google.com/bigquery/docs/row-level-security-intro#how_row-level_security_works) ([configured here](https://github.com/cal-itp/data-infra/blob/main/warehouse/macros/create_row_access_policy.sql)) in the warehouse code to limit access to data to authorized parties this is a multi-step process.
```